src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,124 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.Scanner;
+
+/**
+ * Represents the Ui of Duke.
+ */
+public class Ui {
+    public static String LINE = "____________________________________________________________";
```

Comment:
I like the use of abstracting out the formatting. Also, shouldn't a final keyword be in here, give the all caps variable name ?

Reply:
Hi @pss-coder! Thank you for reviewing my PR and giving me valuable feedback, I did not realise that I had so many naming conventions error, I will go and fix them. Thanks once again.